_data/sidebars/home_sidebar.yml

```diff
@@ -82,6 +82,9 @@ entries:
       url: /upgrading/
       output: web
 
+    - title: Integrate Firebase
```

Comment:
@sethladd It's better to use a short title here since this string appears in the left-hand sidenav (easier to scan).